README.adoc

```diff
@@ -40,20 +40,19 @@ Open Liberty. If you run the application, Maven creates an Open Liberty server a
 the application on it. The application displays a simple web page with a link that, when 
 clicked, calls the servlet to return a simple response of `Hello! How are you today?`.
 
-One benefit of using Maven is that, once you have defined the details of the project and 
-any dependencies it has, Maven automatically handles downloading and installing any and 
-all of the dependencies.
+Using a build tool like Maven is beneficial as you can define the details of the projects and its dependencies.
+The build tool will automatically handle downloading and installing any and all of the dependencies.
+Build tools also allow for defining automated tests that can be easily re-run on each build. 
+If the tests don't pass after changes are made, the build fails so you know you need to fix your code. 
 
-Another benefit of using Maven is that it can run automated tests on the application after 
-building it. You could, of course, test your application manually by starting a server 
-and pointing a web browser at the application URL. Automated tests are a much better 
-approach though because Maven can easily re-run the same tests each time it builds the 
-application. If the tests don't pass after you've made a change to the application, the 
-build fails so you know that you need to fix your code.
+Choosing a build tool often comes down to personal preference, but Maven has some benefits.  
```

Comment:
"Choosing a build tool often comes down to personal preference, but you might choose to use Maven for several reasons. Maven defines its builds using XML, which is probably familiar to you already. As a mature, commonly used build tool, Maven probably integrates with whichever IDE you prefer to use. Maven also has an extensive plugin library which offers a variety of ways to quickly customize your build. So Maven can be a good choice for smaller projects."

I've edited it slightly to match the edited Gradle equivalent. I've also added a little bit about it using XML (see if you agree with the reasoning). On the last point about when it's a good choice - is that true or will we just get people immediately thinking "well I've happily used it in a large project"? Also, would a reason for it being a "good choice" be just familiarity if that's what they're using already (which is likely if they're using any build tool, I think?).

Reply:
> On the last point about when it's a good choice - is that true or will we just get people immediately thinking "well I've happily used it in a large project"?

I think this is a good point. I replaced the last point with a statement saying Maven is a good choice if your team is already familiar with it. Let me know what you think.